tests/TestCase/ORM/TableRegistryTest.php

```diff
@@ -320,6 +321,41 @@ public function testConfigAndBuild()
         $this->assertEquals('users', $table->alias());
         $this->assertSame($connection, $table->connection());
         $this->assertEquals(array_keys($schema), $table->schema()->columns());
+        $this->assertEquals($schema['id']['type'], $table->schema()->column('id')['type']);
+    }
+
+    /**
+     * Tests that table options can be pre-configured with a single validator
+     *
+     * @return void
+     */
+    public function testConfigWithSingleValidator()
+    {
+        TableRegistry::config('users', ['validator' => new Validator()]);
+        $table = TableRegistry::get('users');
+
+        $this->assertInstanceOf('Cake\Validation\Validator', $table->validator('default'));
```

Comment:
This test would pass even before your changes. You should check that the validator that went in, is the same validator that comes out with `assertSame()`.